components/common/src/types/listen_ctl_addr.rs

```diff
@@ -20,9 +20,9 @@ use std::{fmt,
           result,
           str::FromStr};
 
-use super::env_config::EnvConfig;
-use crate::error::{Error,
-                   Result};
+use crate::{error::{Error,
+                    Result},
+            hcore::env::Config as EnvConfig};
```

Comment:
Can we replace

```rust
use crate::hcore::{...};
```
with
```rust
use habitat_core::{...};
```
?

The latter means that we'll eventually be able to remove the superfluous code in `lib.rs` in this crate, and others:
https://github.com/habitat-sh/habitat/blob/0764d62ff5f88b054096bd0867d0aa9a85c3cb63/components/common/src/lib.rs#L16

It also means that we'll be able to more easily see what code is actually coming from this crate versus from another.

(This also applies to any other uses of this pattern, naturally.)

Reply:
I'll take a look!

Reply:
How's this? https://github.com/habitat-sh/habitat/pull/6221/commits/41b3ee5604eb51a335962d180912da4d3b5e9103